src/helper.ts

```diff
@@ -0,0 +1,16 @@
+export const arrayContainsEntries = <T>(collection: T[]) =>
+  Array.isArray(collection) && collection.length > 0;
+
+// TODO: add unit tests
+// TODO: Better naming for this?
+// Cover undefined and null cases, returns 0 in that case
+export const safeArrayLength = <T>(collection: T[]) =>
+  collection ? collection.length : 0;
+
+// https://stackoverflow.com/questions/5999998/check-if-a-variable-is-of-function-type
+export const isFunction = (v: any): v is Function => typeof v === 'function';
+
+export const isLastIndexInArray = <T>(index: number, array: T[]) =>
+  index === safeArrayLength(array) - 1;
+
+export const isNorUndefinedOrNull = (v: any) => !(v === void 0 || v === null);
```

Comment:
It's better to create the function with positive meaning to avoid double negation like:

`!isNorUndefinedOrNull`

Reply:
Agreed, replaced the function:

````
typescript
export const isUndefinedOrNull = (v: any) => v === void 0 || v === null;
```